src/context.rs

```diff
@@ -1 +1,33 @@
-pub struct Context {}
+use crate::event::Event;
+use crate::AResult;
+use chrono::{DateTime, Utc};
+use serde::Serialize;
+use std::thread;
+use std::cell::RefCell;
+
+#[derive(Debug, Clone)]
+pub struct Context {
+    pub key: String,
+    pub context_id: String,
+    pub frame: Frame,
```

Comment:
Must this be pub?

Reply:
need not to be. I'll change. i'll use methods on context to modify or access.